rollup.config.js

```diff
@@ -10,7 +9,6 @@ const plugins = [
     tsconfig: path.resolve(BASE_PATH, 'tsconfig.build.json'),
     include: ['src/**/*.{ts,tsx}'],
   }),
-  terser(),
```

Comment:
no more terser?

Reply:
Well, since this is open-source anyway, I don't think we need to minify the code, do we?
The consumer can do it.

Reply:
But why'd we remove terser? Is that a sensible thing to do in _this_ PR?

Reply:
Reverted.